Makefile

```diff
@@ -22,8 +22,8 @@ all: lz_header.bin
 # unreferenced section ahead of .text, in which case link.lds needs adjusting.
 lz_header.bin: lz_header Makefile
 	objcopy -O binary -S --pad-to 0x10000 $< $@
```

Comment:
This change probably wants splitting into a few separate patches.

First of all, introducing _sl_end means that this --pad-to can be dropped.  There should be nop need to zero-extend the binary when the _sl_header changes 0xffff to _sl_end.

Reply:
I was afraid that GRUB would allocate what's left of SLB for something else, but even in this case, that wouldn't break anything. Pad dropped.

As a side note, it seems that the last byte of SLB cannot be measured by SKINIT even with the maximum size specified in the header.